client/components/Overview/components/Sizes.jsx

```diff
@@ -4,19 +4,31 @@ import React from 'react';
 import SizesRender from './SizesRender';
 
 //
-const Sizes = ({ currentStyle, setCurrentSize, setCurrentQuantity }) => {
+const Sizes = ({
```

Comment:
Low-key wondering why ESLint is throwing a fit about this. It did the same for me!

Reply:
its the way it wants it formatted when the line gets too long